aspnet/fundamentals/logging/sample/src/TodoApi/project.json

```diff
@@ -6,17 +6,16 @@
     "preserveCompilationContext": true
   },
   "dependencies": {
-    "Microsoft.AspNetCore.Mvc": "1.0.0-rc2-final",
-    "Microsoft.AspNetCore.StaticFiles": "1.0.0-rc2-final",
-    "Microsoft.AspNetCore.Server.Kestrel": "1.0.0-rc2-final",
-    "Microsoft.Extensions.Logging": "1.0.0-rc2-final",
-    "Microsoft.Extensions.Logging.Console": "1.0.0-rc2-final",
-    "Microsoft.Extensions.Logging.TraceSource": "1.0.0-rc2-final",
-    "Microsoft.AspNetCore.Server.IISIntegration": "1.0.0-rc2-final",
-    "Microsoft.Extensions.Logging.Filter": "1.0.0-rc2-final",
-    "System.Diagnostics.TextWriterTraceListener": "4.0.0-beta-23516"
+    "Microsoft.AspNetCore.Mvc": "1.0.0",
+    "Microsoft.AspNetCore.StaticFiles": "1.0.0",
+    "Microsoft.AspNetCore.Server.Kestrel": "1.0.0",
+    "Microsoft.Extensions.Logging": "1.0.0",
+    "Microsoft.Extensions.Logging.Console": "1.0.0",
+    "Microsoft.Extensions.Logging.TraceSource": "1.0.0",
+    "Microsoft.AspNetCore.Server.IISIntegration": "1.0.0",
+    "Microsoft.Extensions.Logging.Filter": "1.0.0",
+    "System.Diagnostics.TextWriterTraceListener": "4.0.0-rc2-24027"
```

Comment:
Everything will be RTM at RTM time 😄 At least, certainly everything in CoreFX and ASP.NET. So this doesn't look right.